test/circuit/library/test_raw_feature_vector.py

```diff
@@ -115,7 +116,7 @@ def test_usage_in_vqc(self):
 
         vqc.fit(X, y)
         score = vqc.score(X, y)
-        self.assertGreater(score, 0.5)
+        self.assertGreaterEqual(score, 0.5)
```

Comment:
Why is `greaterEqual`, I guess it is hard to hit in the test `0.5` sharp?

Reply:
Answered above, I will revert to the old check.